src/protagonist/API.Tests/Integration/DeliveryChannelTests.cs

```diff
@@ -0,0 +1,532 @@
+    {
+        // Arrange
+        const int customerId = 88;
+        const string newDeliveryChannelPolicyJson = @"{
+            ""name"": ""post-invalid-policy"",
+            ""displayName"": ""Invalid Policy"",
+            ""policyData"": ""[\""audio-mp3-128\""]""
+        }";
+        
+        var path = $"customers/{customerId}/deliveryChannelPolicies/foo";
+
+        // Act
+        var content = new StringContent(newDeliveryChannelPolicyJson, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(customerId).PostAsync(path, content);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+    
+    [Fact]
+    public async Task Post_DeliveryChannelPolicy_400_IfNameInvalid()
+    {
+        // Arrange
+        const int customerId = 88;
+        const string newDeliveryChannelPolicyJson = @"{
+            ""name"": ""foo bar"",
+            ""displayName"": ""Invalid Policy"",
+            ""policyData"": ""[\""audio-mp3-128\""]""
+        }";
+        
+        var path = $"customers/{customerId}/deliveryChannelPolicies/iiif-av";
+
+        // Act
+        var content = new StringContent(newDeliveryChannelPolicyJson, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(customerId).PostAsync(path, content);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+    
+    [Theory]
+    [InlineData("")] // No PolicyData specified
+    [InlineData("[]")] // Empty array
+    [InlineData("[\"\"]")] // Array containing an empty value
+    [InlineData(@"[\""foo\"",\""bar\""]")] // Invalid data
+    [InlineData(@"[\""100,100\"",\""200,200\""")]  // Invalid JSON
+    public async Task Post_DeliveryChannelPolicy_400_IfThumbsPolicyDataInvalid(string policyData)
+    {
+        // Arrange
+        const int customerId = 88;
+        
+        var newDeliveryChannelPolicyJson = $@"{{
+            ""name"": ""post-invalid-thumbs"",
+            ""displayName"": ""Invalid Policy (Thumbs Policy Data)"",
+            ""policyData"": ""{policyData}""
+        }}";
+        var path = $"customers/{customerId}/deliveryChannelPolicies/thumbs";
+
+        // Act
+        var content = new StringContent(newDeliveryChannelPolicyJson, Encoding.UTF8, "application/json");
+        var response = await httpClient.AsCustomer(customerId).PostAsync(path, content);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+    
+    [Theory]
+    [InlineData("")] // No PolicyData specified
+    [InlineData("[]")] // Empty array
+    [InlineData("[\"\"]")] // Array containing an empty value
+    [InlineData(@"[\""transcode-policy-1\"",\""\""]")] // Invalid data
+    [InlineData(@"[\""transcode-policy\""")] // Invalid JSON
+    public async Task Post_DeliveryChannelPolicy_400_IfAvPolicyDataInvalid(string policyData)
```

Comment:
Think there's an ability to generate a `409` on the `POST` - might have missed it, but I can't see a test for this and would probably be worth adding